### DISCUSSION TITLE
Issue with ARC and numpy 2.0

### DISCUSSION THREAD

**Discussion by SimonSchroers** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#174] [General]
Hello, 
I tried to install the latest numpy version and ARC on my machine. However when initializing the atom type the following error was produced: 

Traceback (most recent call last):
  File "c:\*****", line 18, in <module>
    atom = arc.Rubidium()
           ^^^^^^^^^^^^^^
  File "C:\Users\******\arc\alkali_atom_functions.py", line 221, in __init__
    from .arc_c_extensions import NumerovWavefunction
ImportError: numpy.core.multiarray failed to import

I was able to resolve this by downgrading to numpy version 1.26. Is ARC so far not working for numpy version 2.0 or is there another reason for that error?

**Comment by nikolasibalic** [comment]
Hello, the new Numpy had a major release less than a month ago after years. So very likely we need to double check that things work, and at very least we will need to create wheels compatible with the new Numpy. Please stick to 1.* Numpy with ARC for a time being.

Le 15 juil. 2024 à 17:32, SimonSchroers ***@***.***> a écrit :

﻿

Hello,
I tried to install the latest numpy version and ARC on my machine. However when initializing the atom type the following error was produced:

Traceback (most recent call last):
File "c:*****", line 18, in
atom = arc.Rubidium()
^^^^^^^^^^^^^^
File "C:\Users******\arc\alkali_atom_functions.py", line 221, in init
from .arc_c_extensions import NumerovWavefunction
ImportError: numpy.core.multiarray failed to import

I was able to resolve this by downgrading to numpy version 1.26. Is ARC so far not working for numpy version 2.0 or is there another reason for that error?

**Comment by nikolasibalic** [comment]
Numpy 2.0 should be supported in ARC v3.6.0 (thanks to #179).
